src/pages/workspaces/workspace/notebooks/modals/CloudEnvironmentModal.js

```diff
@@ -85,28 +87,47 @@ export const CloudEnvironmentModal = ({
       onDismiss()
     }
   })
+  const renderGalaxyModal = () => renderAppModal(GalaxyModalBase, NEW_GALAXY_MODE)
+  const renderCromwellModal = () => renderAppModal(CromwellModalBase, NEW_CROMWELL_MODE)
 
-  const renderDefaultPage = () => div({ style: { display: 'flex', flexDirection: 'column', flex: 1 } }, [
-    renderToolButtons(tools.Jupyter.label),
-    renderToolButtons(tools.RStudio.label),
-    renderToolButtons(tools.galaxy.label)
-  ])
+  const renderDefaultPage = () => div({ style: { display: 'flex', flexDirection: 'column', flex: 1 } },
+    _.map(tool => renderToolButtons(tool.label))(getToolsToDisplay)
```

Comment:
This is the sort of change that I'd like the code to move incrementally towards… removing case statements/listing of individual tools and instead iterating over tools to display their elements.

Some examples of things that could easily be moved into the tools object is tool images and editor default sizes.